wagtail/admin/widgets/slug.py

```diff
@@ -6,10 +6,11 @@ class SlugInput(widgets.TextInput):
     def __init__(self, attrs=None):
         default_attrs = {
             "data-controller": "w-slug",
-            "data-action": "blur->w-slug#slugify",
+            "data-action": "blur->w-slug#slugify w-sync:check->w-slug#compare w-sync:apply->w-slug#urlify:prevent",
             "data-w-slug-allow-unicode-value": getattr(
                 settings, "WAGTAIL_ALLOW_UNICODE_SLUGS", True
             ),
+            "data-w-slug-mode-param": "urlify",
```

Comment:
```suggestion
```

Let's remove this completely, we can default to the behaviour we want and that's good enough.